contracts/migration/LiquidityMigrator.sol

```diff
@@ -2,92 +2,110 @@
 pragma solidity 0.7.5;
 
 import "../interfaces/IERC20.sol";
-import "../libraries/SafeERC20.sol";
-import "../libraries/SafeMath.sol";
 import "../interfaces/IUniswapV2Router.sol";
 import "../interfaces/IOlympusAuthority.sol";
-import "../types/OlympusAccessControlled.sol";
 import "../interfaces/ITreasury.sol";
-
-interface IMigrator {
-    enum TYPE {
-        UNSTAKED,
-        STAKED,
-        WRAPPED
-    }
-
-    // migrate OHMv1, sOHMv1, or wsOHM for OHMv2, sOHMv2, or gOHM
-    function migrate(
-        uint256 _amount,
-        TYPE _from,
-        TYPE _to
-    ) external;
-}
+import "../interfaces/IOHM.sol";
+import "../interfaces/IsOHM.sol";
+import "../types/OlympusAccessControlled.sol";
+import "../libraries/SafeERC20.sol";
 
 contract LiquidityMigrator is OlympusAccessControlled {
     using SafeERC20 for IERC20;
-    using SafeMath for uint256;
 
     ITreasury internal immutable oldTreasury = ITreasury(0x31F8Cc382c9898b273eff4e0b7626a6987C846E8);
     ITreasury internal immutable newTreasury = ITreasury(0x9A315BdF513367C0377FB36545857d12e85813Ef);
-    IERC20 internal immutable oldOHM = IERC20(0x383518188C0C6d7730D91b2c03a03C837814a899);
+    IOHM internal immutable oldOHM = IOHM(0x383518188C0C6d7730D91b2c03a03C837814a899);
     IERC20 internal immutable newOHM = IERC20(0x64aa3364F17a4D01c6f1751Fd97C2BD3D7e7f1D5);
-    IMigrator internal immutable migrator = IMigrator(0x184f3FAd8618a6F458C16bae63F70C426fE784B3);
+    IsOHM internal immutable oldsohm = IsOHM(0x04F2694C8fcee23e8Fd0dfEA1d4f5Bb8c352111F);
+    IsOHM internal immutable newsohm = IsOHM(0x04906695D6D12CF5459975d7C3C03356E4Ccd460);
 
     constructor(IOlympusAuthority _authority) OlympusAccessControlled(_authority) {}
 
-    /**
-     * @notice Migrate LP and pair with new OHM
-     */
-    function migrateLP(
-        address pair,
-        IUniswapV2Router routerFrom,
-        IUniswapV2Router routerTo,
-        address token,
-        uint256 _minA,
-        uint256 _minB,
-        uint256 _deadline
+    function remove(
```

Comment:
In which scenarios do we need this? 
Wouldn't we want to migrate only?

Reply:
i had considered proposing the removal of this liquidity rather than migration, since LPs currently make up an unprecedented % of total supply. if said proposal does not occur then yes, unnecessary